index.js

```diff
@@ -108,11 +109,21 @@ function logError(context, error){
   const sonarComponent = argv.sonarcomponent;
   const withOrganization = data.sonarOrganization ? `&organization=${data.sonarOrganization}` : '';
   var headers = {};
+  var version = null;
+  
+  {
+    const res = request(
```

Comment:
sync-request doesn't support proxies. it would be better if all the code that is used to reach sonarqube uses the same library.